assign_nodes.py

```diff
@@ -24,40 +26,91 @@ def unique_nodes(examples):
 
 def find_node(node, kg, ontology = ""):
 	nodes = kg.labels_all
+	### All caps input is probably a gene or protein. Either search in a case sensitive manner or assign to specific ontology. 
+
 
 	if ontology == "":
```

Comment:
Do you use a specific ontology to restrict the search or just making that available here if we want to in the future?

Reply:
Good catch. Not at this time, but thought it could be a future feature.

Reply:
What you could do is call in the search_nodes function with an ontology (gene/PR/GO) and put that logic there so it isn't part of the find_node function. That way it's more clear in what case we want to limit to only search those 3 ontologies (and we could add more cases like that). 

On that note, is it safe to assume upper case nodes will always be one of those? I hadn't thought about it but wondering if that has edge cases

Reply:
yeah, that's what I would do manually. Eventually thought someone will want to include the other ontologies. Would it be better for the user to always choose which ontology to search through?

Also, I don't know. APP was returning thousands of hits since I was not searching in a case-sensitive manner. So that's why I used that logic, though it may be a false assumption.

Reply:
Not sure, that's a good idea though. For now maybe we leave it as is and log that as a future addition? Would make the performance better I bet

Reply:
Yeah. I'll add it to issues.